README.md

```diff
@@ -0,0 +1,1186 @@
+# node-diacritics-transliterator
+
+> Diacritic transliteration tools using diacritics.io API
+
+# Installation
+
+```bash
+$ npm install diacritics-transliterator
+```
+
+# Usage
+
+```js
+const diacritics = require("diacritics-transliterator");
+const de = diacritics.getData({
+    language: "German"
+});
+/* de => {"de":{...}} */
+```
+
+# API
+
+## Contents
+
+- [Initializing](#initializing)
+- [Basic functions](#basic-functions)
+  - [getVersion](#getversion)
+  - [setVersion](#setversion)
+  - [getData](#getdata)
+  - [getDiacritics](#getdiacritics)
+  - [formatUnicode](#formatunicode)
+- [Transliteration functions](#transliteration-functions)
+  - [transliterate](#transliterate)
+  - [createRegExp](#createregexp)
+  - [replacePlaceholder](#replaceplaceholder)
+- [Command Line Interface](#command-line-interface)
+
+## Initializing
+
+Each API method requires the `diacritics-transliterator` module.
+
+**Syntax**
+
+```js
+const diacritics = require("diacritics-transliterator");
+```
+
+**Return Value**
+
+The diacritics module.
+
+**Parameters**
+
+<dl>
+  <dt><code>version</code> (optional)</dt>
+  <dd>The version of the database API to use. Defaults to the current version.</dd>
```

Comment:
Oh, there aren't any parameters. I need to remove that. You set the version like this:

```js
const diacritics = require("diacritics-transliterator");
diacritics.version = "v1";
```

You can find this under basic functions > setVersion